utils.py

```diff
@@ -0,0 +1,25 @@
+import sys
```

Comment:
Instead of utils.py can you change the file name to something like `review_board_Exceptions.py`? 
@bhavin192 what is your thougth on that?